pkg/probe/log_fortianalyzer_queue.go

```diff
@@ -0,0 +1,55 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaQueueResults struct {
+	Connected  int `json:"connected"`
+	FailedLogs int `json:"failed_logs"`
+	CachedLogs int `json:"cached_logs"`
+}
+
+type LogAnaQueue struct {
+	Results LogAnaQueueResults `json:"results"`
+	VDOM    string             `json:"vdom"`
+	Version string             `json:"version"`
+}
+
+func probeLogAnalyzerQueue(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		logAnaConn = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_queue_connected",
```

Comment:
```suggestion
			"fortigate_log_fortianalyzer_queue_connections",
```
We use `connections` at the most metric endpoints